### PR TITLE
update agenda id's

### PR DESCRIPTION
**Pull request by hatch01**
see https://github.com/onyx-lyon1/lyon1-agendaclonner for details

### CODE REVIEW

File: apps/onyx/lib/screens/agenda_config/states/agenda_config_cubit.dart [hunk 1]
```diff
@@ -20,10 +20,11 @@ class AgendaConfigCubit extends Cubit<AgendaConfigState> {
...
     emit(state.copyWith(status: AgendaConfigStatus.loading));
     try {
-      dirs = await compute(
-          AgendaConfigLogic.loadDirs,
-          LoadData(await rootBundle.loadString('assets/agenda_ids.json.enc'),
-              await rootBundle.loadString('assets/key.txt')));
+      dirs = await compute(AgendaConfigLogic.loadDirs, (
+        encryptedData:
+            await rootBundle.loadString('assets/agenda_ids.json.enc'),
+        key: await rootBundle.loadString('assets/key.txt')
```
Comment (Persson-dev) [inline apps/onyx/lib/screens/agenda_config/states/agenda_config_cubit.dart:26]: Use constants

File: .gitmodules
Comment (Persson-dev) [inline .gitmodules:1]: Do we need to clone it to build Onyx, or is it only used by FDroid ?

**Review by Persson-dev — COMMENTED** [review]
Some minor comments

File: .gitmodules
Comment (hatch01) [reply .gitmodules:1]: it's only for Fdroid

File: apps/onyx/lib/screens/agenda_config/states/agenda_config_cubit.dart (hunk 1 above)
Comment (hatch01) [reply apps/onyx/lib/screens/agenda_config/states/agenda_config_cubit.dart:26]: This is the only usage of this. I don't think it's very useful

Comment (Persson-dev) [reply apps/onyx/lib/screens/agenda_config/states/agenda_config_cubit.dart:26]: Even if you use it once, it's better to not introduce some random strings in the code

**Comment by Persson-dev** [comment]
Merged with command line

**Pull request closed without merging**